### PR TITLE
[zoom] - Update package-spec to 2.9.0

### PR DESCRIPTION
**Pull request by taylor-swanson**
# What does this PR do?

- Updates the package-spec version to `2.9.0`
- Fixed incorrect if statement when setting `event.type`

```
[git-generate]
go run github.com/andrewkroh/go-examples/ecs-update@latest -ecs-version=8.8.0 -ecs-git-ref=8.8 -format-version=2.9.0 packages/zoom
```

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Related issues

- Relates elastic/security-team#5870

### CODE REVIEW

File: packages/zoom/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.13.0"
+  changes:
+    - description: Update package to pkg-spec 2.9.0.
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/6675
```
Comment (bhapas) [inline packages/zoom/changelog.yml:6]: Description of the `event.type` bugfix too

**Comment by taylor-swanson** [comment]
Looks like elastic-package has not been updated yet, waiting on #6679

**Review by efd6 — APPROVED** [review]

**Pull request merged**